json-common/src/main/java/com.cerner.beadledom.json.common/models/ErrorDetail.java

```diff
@@ -0,0 +1,88 @@
+package com.cerner.beadledom.json.common.models;
```

Comment:
Should be singular.

Reply:
Changed the package name from `models` to `model`: https://github.com/cerner/beadledom/pull/45/commits/5f20764db12495ac1e321368a52939c7ce24444d